docs/en-us/workloadspread.md

```diff
@@ -3,79 +3,114 @@ title: WorkloadSpread
 ---
 # WorkloadSpread
 
-WorkloadSpread can be used to constrain the spread of stateless workload, which empower single workload the abilities for
-multi-domain deploy and elastic deploy.
+**FEATURE STATE:** Kruise v0.10.0
 
-The feature of WorkloadSpread is similar with UnitedDeployment in Kruise community. Each WorkloadSpread defines multi-domain
+WorkloadSpread can distribute Pods of workload to different types of Node according to some rules, which empowers single workload the abilities for
+multi-domain deployment and elastic deployment.
+
+Some common rules include:
+- Horizontal spread (for example, average spread in dimensions such as host, az, etc.)
+- Spread according to the specified ratio (for example, deploy Pod to several specified az according to the proportion)
+- Different subset management with priority, such as
```

Comment:
consider add another policy 
- subset management with customization, such as
   - control how many pods in a workload are deployed in different cpu arch
   - enable pods in different cpu arch to have different resource requirements